test/unit/specs/util/loanUtils.spec.js

```diff
@@ -154,4 +154,40 @@ describe('loanUtils.js', () => {
 			expect(result).toEqual(['Agriculture']);
 		});
 	});
+	describe('DropdownPrices', () => {
```

Comment:
The describe usually just mentions the tested method, so `getDropdownPriceArray` here.